### ISSUE TITLE
Use quotes in another project

### ISSUE THREAD

**Issue by hpehl** [skolakoda/programming-quotes-api#53]
Hi there, 

I came across your project while looking for quotes I can use in an example I'm building. Since I could not find a license, I'd like to ask you for permission to use `quotes.json` in my project. 

Thanks!

**Comment by mudroljub** [comment]
Yes, of course, the project is open source. Good luck with your project!

**Comment by xamidi** [comment]
@mudroljub
https://opensource.guide/legal/#why-do-people-care-so-much-about-the-legal-side-of-open-source
